token-client/src/main/java/com/sap/cloud/security/xsuaa/client/OAuth2TokenKeyService.java

```diff
@@ -16,31 +16,48 @@
 public interface OAuth2TokenKeyService {
 
 	/**
-	 * Same as {@link #retrieveTokenKeys(URI, String)} except that zoneId is set to
-	 * {@code null}.
-	 *
 	 * @deprecated gets removed in favor of {@link #retrieveTokenKeys(URI, String)}
 	 *             with next major version 3.0.0
 	 */
 	@Deprecated
 	default String retrieveTokenKeys(@Nonnull URI tokenKeysEndpointUri) throws OAuth2ServiceException {
 		throw new UnsupportedOperationException(
-				"for security reason this internal method was replaced with retrieveTokenKeys(<jwks_url>, <zone_uuid>)");
+				"for security reason this internal method was replaced with retrieveTokenKeys(<jwks_url>, <app_tid>)");
 	}
 
 	/**
-	 * Requests token web key set from OAuth Server.
+	 * Requests token web key set from Xsuaa OAuth Server.
```

Comment:
Remove Xsuaa from this line if it is also used for IAS?

Reply:
this would be used only for Xsuaa as it doesn't require client id for IAS then client id needs to be provided as well

Reply:
Okay, let's adjust the Javadoc for the "tenantId" parameter then. I mentioned it in the other comment because it is suddenly talking about IAS.